frontend/src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeaderMultipleValues.tsx

```diff
@@ -26,14 +25,38 @@ interface ConstraintSingleValueProps {
     allowExpand: (shouldExpand: boolean) => void;
 }
 
+const StyledHeaderValuesContainerWrapper = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'stretch',
+    margin: 'auto 0',
+}));
+
+const StyledHeaderValuesContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'stretch',
+    margin: 'auto 0',
+    flexDirection: 'column',
+    marginLeft: theme.spacing(1),
+    [theme.breakpoints.down(710)]: {
```

Comment:
I think the value here should be extracted to avoid "magic number" issue in the future.